process/RequirementsForAdditionalFunctionality.md

```diff
@@ -0,0 +1,47 @@
+# Requirements for additional functionality to WebGPU
+
+This is a list of requirements which proposals for new functionality must adhere to in order to be standardized. New functionality is expected to usually take the form of a new [feature](https://gpuweb.github.io/gpuweb/#feature-index) to WebGPU.
+
+Adhering to these requirements is _necessary but not sufficient_ for proposals to be standardized by the WebGPU Community Group / Working Group. A proposal which meets these requirements may still be rejected by the CG / WG for other reasons.
+
+The list is expected to grow (or maybe even shrink!) over time. This list represents the CG's / WG's current processes. The initial version of this list was taken mostly from [this GitHub thread](https://github.com/gpuweb/gpuweb/issues/2310).
+
+## Glossary
+
+- "Browser engine" means "(Blink and Dawn) / (Gecko and wgpu (and maybe more?)) / (WebKit and WebGPU.framework) / etc."
```

Comment:
We can remove the "and maybe more?" bit, since there is `/etc` at the end.
Were you thinking of Servo or Deno here?

Reply:
Oh, no, I was thinking that Gecko seems to be using a constellation of frameworks/libraries inside it which all contribute to WebGPU. That constellation, taken together, should be considered a single browser engine.

Reply:
ah, ok. Describing it as "Gecko and wgpu" is totally fair. The only other significant thing in the constellation is "naga". Whether it's listed here or not doesn't matter too much, but should be consistent with listing Tint alongside of Dawn.